tests/lib/rules/space-before-keywords.js

```diff
@@ -218,25 +218,6 @@ ruleTester.run("space-before-keywords", rule, {
             code: "; class Bar extends Foo.Baz {}",
             ecmaFeatures: { classes: true },
             options: never
-        },
-        // Super
-        {
```

Comment:
Do you mean just the valid ones, or all of them? They valid ones still pass, but it didn't make much sense to me keeping them after removing the check on super.